.github/workflows/try.yml

```diff
@@ -78,7 +78,7 @@ jobs:
             // Process `workflow_dispatch` provided configuration overrides.
             if (context.eventName == "workflow_dispatch") {
               // WPT-related overrides only affect Linux currently, as tests don't run by default on other platforms.
-              configuration.matrix[0].wpt_layout = "${{ inputs.wpt-layout }}" || "none";
+              configuration.matrix[0].wpt = "${{ inputs.wpt }}" || "none";
```

Comment:
The parameter `wpt-layout` in line 17 of this file also needs to be changed to a boolean `wpt`. 

And to match that change, this particular line should assign a boolean value to `configuration.matrix[0].wpt` instead of a string (and we shouldn't fallback to "none")
 i.e we should do `configuration.matrix[0].wpt = Boolean(${{ inputs.wpt }})`

Reply:
@MDCODE247 the RHS of the assignment here should also be changed as explained above.

> And to match that change, this particular line should assign a boolean value to configuration.matrix[0].wpt instead of a string (and we shouldn't fallback to "none")
i.e we should do configuration.matrix[0].wpt = Boolean(${{ inputs.wpt }})